cmd/thanos/query.go

```diff
@@ -126,6 +128,7 @@ func registerQuery(app *extkingpin.App) {
 	defaultEvaluationInterval := extkingpin.ModelDuration(cmd.Flag("query.default-evaluation-interval", "Set default evaluation interval for sub queries.").Default("1m"))
 
 	storeResponseTimeout := extkingpin.ModelDuration(cmd.Flag("store.response-timeout", "If a Store doesn't send any data in this specified duration then a Store will be ignored and partial data will be returned if it's enabled. 0 disables timeout.").Default("0ms"))
+	reqLogConfig := *extkingpin.RegisterRequestLoggingFlags(cmd)
```

Comment:
Something is wrong. I don't think we want value - we need to keep pointer if you want this flag to work (:

Reply:
My bad :)